src/engraving/internal/engravingfont.cpp

```diff
@@ -852,8 +852,8 @@ void EngravingFont::computeMetrics(EngravingFont::Sym& sym, const Smufl::Code& c
     }
 
     if (sym.code > 0) {
-        sym.bbox = fontProvider()->symBBox(m_font, sym.code);
-        sym.advance = fontProvider()->symAdvance(m_font, sym.code);
+        sym.bbox = fontProvider()->boundingRect(m_font, sym.code);
+        sym.advance = fontProvider()->horizontalAdvance(m_font, sym.code);
```

Comment:
_Maybe_ (part of) the reason was that that `sym.code` doesn't fit in `Char` for all symbols but needs `char32_t` instead?

Reply:
So, about the first point: I ended up generalizing all the metric functions to 32bit characters, cause there's really no reason not to. That way we can also get rid of the inFont vs inFontUcs4 distinction.

About the second point: yes the Smufl code points are all within the 16 bit range, however I discovered that the unicode standard also has a range dedicated to music symbols which is beyond the 16 bit range. Smufl suggests that if a symbol exists both in Smufl and in Unicode it should be included at both code points but some of our fonts only included it at the Unicode point, so we do need to keep the 32bit variable after all.